godot-ffi/src/toolbox.rs

```diff
@@ -529,9 +529,15 @@ mod manual_init_cell {
     unsafe impl<T: Send + Sync> Sync for ManualInitCell<T> {}
     // SAFETY: See `Sync` impl.
     unsafe impl<T: Send> Send for ManualInitCell<T> {}
+
+    impl<T> Default for ManualInitCell<T> {
+        fn default() -> Self {
+            Self::new()
+        }
+    }
```

Comment:
I don't think this is necessary, people are unlikely to use the type generically, and I personally prefer keeping APIs smaller. It avoids the question whether a caller should use `new()` or `default()`.

Reply:
Yeah, Clippy was complaining about it after I made the type public. I can also suppress the lint.

Reply:
Moved to `godot-ffi`, so this goes away entirely.